src/main/java/com/linkedin/kmf/services/configs/ProduceServiceConfig.java

```diff
@@ -70,6 +74,11 @@
                                     NewProducer.class.getCanonicalName(),
                                     ConfigDef.Importance.LOW,
                                     PRODUCER_CLASS_DOC)
+                            .define(PARTITIONER_CLASS_CONFIG,
+                                    ConfigDef.Type.STRING,
+                                    DefaultPartitioner.class.getCanonicalName(),
```

Comment:
We probably need to decide the default value based on the producer class (e.g. new or old). If user has explicitly configured both producer class and partitioner class that conflicts with each other, we should probably throw warning and quit. But its OK if you don't do it here. I can do it in another patch.